.github/workflows/test_package.yml

```diff
@@ -23,8 +23,6 @@ jobs:
     - name: Install dependencies
       run: |
         brew install cmake
-    - name: Clone repo
```

Comment:
I would actually put this back, and run the tests from the cloned repo?

Reply:
The only problem with doing this is that the versions will no longer match up. If you are installing from pip, you need to run the tests from that version, otherwise normal development will break this run regularly. 

In general if we aren't going to package tests, I would argue it is better to focus on catching bugs before shipping, and then just make sure the package can be installed and core functionality run. Perhaps we could add the a little to the bash script run instead?